src/components/QueryEditor/visual-query-builder/AwesomeQueryBuilder.tsx

```diff
@@ -25,8 +27,9 @@ const buttonLabels = {
 export const emptyInitValue: JsonItem = {
   id: Utils.uuid(),
   type: 'group' as const,
-  children1: {
-    [Utils.uuid()]: {
+  children1: [
```

Comment:
I followed https://github.com/ukrbublik/react-awesome-query-builder#migration-to-520 here